config/envs/development.py

```diff
@@ -1,11 +1,14 @@
 """Flask Local Development Environment Configuration."""
 import logging
+from os import getenv
 
 from config.envs.default import DefaultConfig
+from distutils.util import strtobool
 from fsd_utils import configclass
 
 
 @configclass
 class DevelopmentConfig(DefaultConfig):
 
     FSD_LOGGING_LEVEL = logging.DEBUG
+    USE_LOCAL_DATA = strtobool(getenv("USE_LOCAL_DATA", "True"))
```

Comment:
I think if one sets USE_LOCAL_DATA in the ENV this will automatically take precedence anyway given the config util

Reply:
Doesn't appear so from my testing, e.g. if I hardcode this to "True" the ENV var is ignored.

Reply:
apologies - I got it the wrong way around then, class config overrides the environment